### ISSUE TITLE
Refactor QuestionsController::view

### ISSUE THREAD

**Issue by davidyell** [davidyell/Learning-CakePHP3#4]
Refactor the find into a number of small model based custom find methods.

https://github.com/davidyell/Learning-CakePHP3/blob/master/src/Controller/QuestionsController.php#L39-L61

**Comment by davidyell** [comment]
This will need two custom find methods for the User and the UserComments. Perhaps a shared find as a trait across both Table classes.

**Comment by davidyell** [comment]
Resolved by #5

**Comment by davidyell** [comment]
This is causing a type error in the query.

```
Warning (4096): Argument 1 passed to App\Model\Table\QuestionsTable::findUserCommentsByCreated() must be an instance of App\Model\Table\Query, instance of Cake\ORM\Query given, called in /home/neon/Sites/CakeExchange/vendor/cakephp/cakephp/src/ORM/Table.php on line 1446 and defined [APP/Model/Table/QuestionsTable.php, line 52]
```

**Comment by davidyell** [comment]
Resolved in aaf836ab239175638456d1a72f5a933dcf5ff106
